File: Data Structures & Algorithms/majority-element-ii/submission-0.py
class Solution:
    def majorityElement(self, nums: List[int]) -> List[int]:
        occurrences = {}
        for num in nums:
            if num not in occurrences.keys():
                occurrences[num] = 1
            else:
                occurrences[num] += 1
        res = []
        for item in occurrences.items():
            if item[1] > len(nums) // 3:
                res.append(item[0])
        return res

File: Data Structures & Algorithms/maximum-subarray/submission-0.cpp
class Solution {
public:
    int maxSubArray(vector<int>& nums) {
        int maxSum= nums[0];
        int curSum= 0;
        for (int num : nums) {
            curSum= max(curSum, 0) + num;
            maxSum= max(curSum, maxSum);
        }
        return maxSum;
    }
};
